notebooks/metagenomics_pipeline_iSeqnorm.ipynb

```diff
@@ -1098,6 +1098,11 @@
    "source": [
     "sequencer = 'iSeq'\n",
     "lanes = [1]\n",
+    "plate_df['vol_extracted_elution_ul'] = 70\n",
+    "# CHARLIE ADDED\n",
+    "plate_df['mass_syndna_input_ng'] = 100\n",
+    "plate_df['extracted_gdna_concentration_ng_ul'] = 100\n",
+    "# END CHARLIE ADDED\n",
```

Comment:
Are these comments needed?

Reply:
I they are for me to clean up.

Reply:
Those columns were not present in plate_df because the notebooks (...iSeqnorm and matrix_tube_pipeline...) default to run without synDNA spike in (syndna_pool_number = None). However, when I do include synDNA spike-in, the notebook runs to completion with no issues.

Reply:
Hmm, this is troublesome since the metadata directs make_sample_sheet() to create an AbsQuantSampleSheetv10() object, which requires those two columns. I don't mind if they're empty, but as of now they do need to be present. I'm confused because it sounds like these two columns are suddenly optional, and I have no information on how to treat them.

Reply:
they will be present in plate_df when the user tries to make a abs_quant_metag samplesheet but won't be present when the user runs a standard_metag workflow. Those columns get added to plate_df by the addsyndna() function but only when synDNA_pool_number != None.